src/modules/battery.cpp

```diff
@@ -1,9 +1,16 @@
 #include "modules/battery.hpp"
-
+#if defined(__FreeBSD__)
+// clang-format off
+#include <sys/types.h>
```

Comment:
Did you try without `<sys/types.h>`? It can be often omitted due to being included by other headers. See also https://github.com/freebsd/freebsd-src/commit/56ee5c551f89

Reply:
Thanks! Indeed, it works without.
`<sys/types.h> removed.